blocks/card/card.js

```diff
@@ -0,0 +1,41 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  const card = document.createElement('div');
+  card.className = 'card-box';
+
+  const row = block.firstElementChild;
+  if (row) {
+    const [cardImage, cardBody, cardOrientation] = row.children;
+
+    if (cardImage) {
+      const img = cardImage.querySelector('img');
+      if (img) {
+        img.className = 'card__image';
+        card.append(img);
+      }
+    }
+
+    if (cardBody) {
+      cardBody.className = 'card__body';
+      const [titleElement, descriptionElement] = cardBody.children;
+      if (titleElement) titleElement.className = 'card__title util-title-s';
+      if (descriptionElement) descriptionElement.className = 'card__description util-body-xs';
+      card.append(cardBody);
+    }
+
+    if (cardOrientation) {
+      const orientation = cardOrientation.textContent.trim();
+      if (orientation === 'horizontal' || orientation === 'vertical') {
```

Comment:
We don't need a horizontal/vertical distinction for the cards - they will always be vertically-oriented (image on top, words on bottom) in a smaller container size, and always horizontally-oriented (image on right, words on left) in a larger container size.

The ticket wasn't written this way, so this is a miss on our part.